File: dev-superior-algoritmo-e-logica-de-programacao/c/exercicios/soma_vetor.c
#include <stdio.h>

int main() {
    double vet[10], soma, media;
    int N;

    printf("Quantos numeros voce vai digitar?");
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        printf("Digite um numero:");
        scanf("%lf", &vet[i]);
    }

    printf("\nVALORES = ");
    for (int i = 0; i < N; i++) {
        printf("%.1lf ", vet[i]);
    }

    soma = 0;
    for (int i = 0; i < N; i++) {
        soma = soma + vet[i];
    }

    printf("\nSOMA = %.2lf", soma);
    media = soma / N;
    printf("\nMEDIA = %.2lf", media);

    return 0;
}

File: dev-superior-algoritmo-e-logica-de-programacao/java/src/exercicios/SomaVetor.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class SomaVetor {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

	    double soma, media;
		int N;

	    System.out.print("Quantos numeros voce vai digitar? ");
		N = sc.nextInt();

	    double[] vet = new double[N];

		for (int i = 0; i < N; i++) {
	    	System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
	    }

		System.out.print("VALORES = ");
		for (int i = 0; i < N; i++) {
			System.out.printf("%.1f  ", vet[i]);
		}

		soma = 0;
		for (int i = 0; i < N; i++) {
			soma = soma + vet[i];
		}

		System.out.printf("\nSOMA = %.2f\n", soma);

		media = soma / N;
	    System.out.printf("MEDIA = %.2f\n", media);

		sc.close();
	}
}
